model/import/README.md

```diff
@@ -3,7 +3,7 @@
 Here you can find instructions about import items using TAO platform.
 
 ## Configuration
-The feature is disabled by default, to enable it flag `FEATURE_FLAG_TABULAR_IMPORT_ENABLED` should be set at the env variables.
+The feature is disabled by default. To enable it, flag `FEATURE_FLAG_TABULAR_IMPORT_ENABLED` should be set to True at the env variables.
```

Comment:
```suggestion
The feature is disabled by default. It can be enabled by the feature flasg `FEATURE_FLAG_TABULAR_IMPORT_ENABLED`.
```

Cause I think how to set feature flag is already described in another doc and we want to avoid duplicate this information.